components/dependency_injection/compilation.rst

```diff
@@ -8,8 +8,8 @@ The service container can be compiled for various reasons. These reasons
 include checking for any potential issues such as circular references and
 making the container more efficient by resolving parameters and removing
 unused services. Also, certain features - like using
-:doc:`parent services </components/dependency_injection/parentservices>` -
-require the container to be compiled.
+:doc:`parent services </components/dependency_injection/parentservices>`
+- require the container to be compiled.
```

Comment:
Nope :) http://pr-5550-6qmocelev2lwe.eu.platform.sh/components/dependency_injection/compilation.html (I love this new platform.sh thing!)